demand-capacity-mgmt-backend/src/main/java/org/eclipse/tractusx/demandcapacitymgmt/demandcapacitymgmtbackend/security/AppSecurityConfigProperties.java

```diff
@@ -0,0 +1,11 @@
+package org.eclipse.tractusx.demandcapacitymgmt.demandcapacitymgmtbackend.security;
```

Comment:
License and Copyright header missing, please refer to [TRG 7.02 - License and Copyright header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02).

Reply:
fixed with https://github.com/eclipse-tractusx/demand-capacity-mgmt/pull/72/commits/5dae4a27379df156672d326210033489a710b240